packages/react-cards/src/components/Card/Card.types.ts

```diff
@@ -24,16 +24,31 @@ export interface ICardSlots {
   stack?: IStackSlot;
 }
 
-// Extending IStyleableComponentProps will automatically add stylable props for you, such as styles and theme.
-//    If you don't want these props to be included in your component, just remove this extension.
 export interface ICardProps extends ICardSlots, IStyleableComponentProps<ICardProps, ICardTokens, ICardStyles>, IBaseProps<ICard> {
   /**
    * Defines whether to render a regular or a compact Card.
    * @defaultvalue false
    */
   compact?: boolean;
+
+  /**
+   * Defines the spacing between Card children.
+   */
+  gap?: number | string;
```

Comment:
David and I have talked previously about how this is something that should be accessible via theming. This is fairly awkward to have to specify as a standalone prop in every component instance. I wonder if `gap` is actually a `token` idea that needs a better name. Maybe `contentPadding`? Hmmmmmmm......

Reply:
Should we consider this as part of this PR or does this need further discussion? In any case, if we actually change `gap` to something else we will have to consider that changing `gap` in `Stack` is a breaking change.

Reply:
Is react-cards still in experimental state? If not, I'd rather consider this as part of this PR rather than set `gap` in stone for Card.

Regarding Stack, I don't think it would have to be a breaking change. Could we not support both and deprecate `gap`?

Reply:
We can do it in this PR, I was just asking. And yeah, I suppose we can deprecate `gap` and support the other approach.

Do you want `contentPadding` to be the token name? Or should we do more brainstorming on the name?